payload-schemas/api.github.com/package/published.schema.json

```diff
@@ -33,138 +33,221 @@
         "ecosystem": { "type": "string" },
         "package_type": {
           "type": "string",
-          "enum": ["npm", "maven", "rubygems", "docker", "nuget", "container"],
+          "enum": ["npm", "maven", "rubygems", "docker", "nuget", "CONTAINER"],
```

Comment:
Can someone verify if all the other values are also in caps?

Reply:
Where are you getting the caps from? I'm looking internally at some GitHub code and I see container is lower-cased, as are the rest of the values.

Reply:
It's the payload examples in https://github.com/octokit/webhooks.net/issues/178